pyo3-ffi/Cargo.toml

```diff
@@ -32,6 +32,8 @@ abi3-py38 = ["abi3-py39", "pyo3-build-config/abi3-py38"]
 abi3-py39 = ["abi3-py310", "pyo3-build-config/abi3-py39"]
 abi3-py310 = ["abi3", "pyo3-build-config/abi3-py310"]
 
+# Automatically generates `python3.dll` import libraries for Windows targets.
+generate-abi3-import-lib = ["pyo3-build-config/python3-dll-a"]
```

Comment:
Wouldn't it be more future proof if the `pyo3` feature would forward to the `pyo3-ffi` feature of the same name instead of also setting the `pyo3-build-config` feature directly?

Reply:
Do we still need to enable `pyo3-build-config/python3-dll-a` directly from `pyo3`?